base_layer/core/src/chain_storage/postgres_db/schema.rs

```diff
@@ -49,7 +49,7 @@ table! {
         features_maturity -> Int8,
         commitment -> Text,
         proof -> Nullable<Bytea>,
-        input -> Text,
+        tx_output -> Nullable<Text>,
```

Comment:
This name is more confusing imho. 

what about `created_in_block`

Reply:
When can UTXOs be null?

Reply:
Agree, better name. Changed. 

But I agree should not be nullable. But are we allowing a transaction to be saved? This means it is required to be nullable.

Reply:
What do you mean by 'saved'?

Reply:
mempool transactions. Are we storing those on disc?

Reply:
Mempool is what is says on the box - in memory

Reply:
cwl, so we never ever store a transaction on disc
Because then that can go out

Reply:
Agreed